es6-destructuring/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>es6-destructuring</title>
+  </head>
+  <body>
+    <!-- html goes here -->
+    <script>
+      const book1 = {
+        title: 'Goodnight Punpun',
+        author: 'Inio Asano',
+        libraryID: 3353
+      };
+      const { title, author, libraryID } = book1;
+      console.log(`The title of the book is ${title}, the author is ${author}, and the library id is ${libraryID}`);
+      const book2 = {
+        title: 'Les Fleurs du mal',
+        author: 'Charles Baudelaire',
+        libraryID: 2345
+      };
+      const { title: titleTwo, author: authorTwo, libraryID: libraryIDTwo } = book2;
+      console.log(`The title of the book is ${titleTwo}, the author is ${authorTwo}, and the library id is ${libraryIDTwo}`);
+      const library = [
+        {
+          title: 'The Road Ahead',
+          author: 'Bill Gates',
+          libraryID: 1254
+        },
+        {
+          title: 'Walter Isaacson',
+          author: 'Steve Jobs',
+          libraryID: 4264
+        },
+        {
+          title: 'Mockingjay: The Final Book of The Hunger Games',
+          author: 'Suzanne Collins',
+          libraryID: 3245
+        },
+        {
+          title: "Dead Dead Demon's De De De De Destruction",
+          author: 'Inio Asano',
+          libraryID: 1233
+        }
+      ];
+      const [book3, book4, book5] = library;
+      const [, book6] = library;
```

Comment:
Here we want to get the fourth element of the array, so you will need a couple more commas to skip the 1st-3rd elements.